home/tests/test_api.py

```diff
@@ -2120,6 +2219,10 @@ def test_assessment_detail_endpoint_filter_by_tag(self, uclient):
         assert meta["slug"] == self.low_result_page.slug
         assert meta["parent"]["id"] == self.low_result_page.get_parent().id
         assert meta["locale"] == "en"
+        assert (
+            meta["detail_url"]
+            == f"http://localhost/api/v2/pages/{self.low_result_page.id}/"
+        )
```

Comment:
Could we maybe have one test that just asserts the entire response? That way we can make sure none of the other meta fields have incorrect values, and we can ensure that future changes don't change the API?

Reply:
Done